go/datas/remote_database_handlers.go

```diff
@@ -88,18 +89,25 @@ func handleWriteValue(w http.ResponseWriter, req *http.Request, ps URLParams, cs
 		io.Copy(ioutil.Discard, reader)
 		reader.Close()
 	}()
-	tc := types.NewTypeCache()
-	vbs := types.NewValidatingBatchingSink(cs, tc)
+	vbs := types.NewValidatingBatchingSink(cs)
 	vbs.Prepare(deserializeHints(reader))
 
-	chunkChan := make(chan *chunks.Chunk, 16)
+	chunkChan := make(chan interface{}, 16)
```

Comment:
what if we make this buffer bigger? like 1024 or something. 16 seems like an odd number. a bigger buffer here means possibly more memory burned with pre-loading chunks off the network, but 64k seems a bit small. what about 256 (==1MB @ 4k/chunk)?